src/grisp_tools_util.erl

```diff
@@ -249,8 +251,17 @@ with_file(File, Opts, Fun) ->
 pipe(State, Actions) ->
     lists:foldl(fun(Action, S) -> Action(S) end, State, Actions).
 
+otp_package_cache() ->
+    cache(package).
+
+find_files(Dir, Regex) ->
+    find_files(Dir, Regex, true).
+
 %--- Internal ------------------------------------------------------------------
 
+find_files(Dir, Regex, Recursive) ->
```

Comment:
This internal version is not needed, right? Since all calls are to `find_files/2` so it could call the `filelib` module directly?

Reply:
yes you are right